### DISCUSSION TITLE
In Depth Longer Running Financials

### DISCUSSION THREAD

**Discussion by masonfields** [voxeltycoon/issues#907] [General]
I would love the income statement to run for several years by month to compare changes overtime. I'll add a new line and often it takes several months to realize profits, which is impossible to see with the current two months. Other in depth financial tools would be greatly appreciated.

**Comment by JeroenDevHero** [comment]
Exactly my big wish! Would make the game absolutely more fun. Since the development of wear is in progress as well, it would also be nice to see this in the financials.
